widget/view.controller.js

```diff
@@ -411,13 +482,27 @@
             if (!CommonUtils.isUndefined(threatHuntConfigForm.fazForm) && threatHuntConfigForm.fazForm.$invalid) {
                 var huntToolIndex = $scope.selectedEnv.huntTools.indexOf('Fortinet FortiAnalyzer');
                 _activeErrorTab('Fortinet FortiAnalyzer', huntToolIndex);
+                loadActiveTab(huntToolIndex, 'Fortinet FortiAnalyzer');
```

Comment:
This is a common code and tabs are checked in a static way. This can be extracted as a common function and make it dynamic for addition of future hunt tools